ts/features/bonus/cgn/store/actions/merchants.ts

```diff
@@ -0,0 +1,13 @@
+import { ActionType, createAsyncAction } from "typesafe-actions";
+import { NetworkError } from "../../../../../utils/errors";
+
+/**
+ * get and handle list of merchants conventioned with CGN
+ */
+export const cgnMerchants = createAsyncAction(
+  "CGN_MERCHANTS_REQUEST",
+  "CGN_MERCHANTS_SUCCESS",
+  "CGN_MERCHANTS_FAILURE"
+)<void, ReadonlyArray<any>, NetworkError>();
```

Comment:
We should avoid to use `any`
I guess it is just a placeholder but if we forget to replace it we can use the action and the reducer with no warnings from compiler.

Could instead possibile to create and use a temporary type?
`type CGNMerchant = unknown;`